.gitignore

```diff
@@ -12,3 +12,9 @@ __pycache__/
 Icon?
 ehthumbs.db
 Thumbs.db
+
+# IDE files
+/.vscode
+
+# Intermediate test files
+out.json
```

Comment:
I would drop this `out.json` line. It is not a general and common file other people are going to have that we want to ignore.

A good `.gitignore` file should be fairly sparse and conservative, because it has the ability to cause surprising behavior, like silently hiding files that you intended to check in. This is particularly true for ignore patterns which aren't anchored to a particular path, because the pattern can match a file anywhere in the source tree.

Reply:
That makes sense. I added it there for development but I definitely would have forgotten about it if you hadn't pointed it out.